conf/available_modules.go

```diff
@@ -8,9 +8,10 @@ package main
 import (
 	_ "mig/modules/agentdestroy"
 	_ "mig/modules/file"
+	_ "mig/modules/migoval"
```

Comment:
nit: s/migoval/oval/ ? or `pkg` ? (I like `pkg`, it's short to type on the command line.